src/smc-webapp/editor_course.cjsx

```diff
@@ -432,6 +432,12 @@ exports.init_redux = init_redux = (redux, course_project_id, course_filename) ->
             title = "#{store.get_student_name(student_id)} - #{store.getIn(['settings', 'title'])}"
             redux.getActions('projects').set_project_title(student_project_id, title)
 
+        start_all_student_projects: () =>
+            get_store()?.get_students().map (student, student_id) =>
+                student_project_id = student.get('project_id')
+                if student_project_id?
+                    redux.getActions('projects').start_project(student_project_id)
```

Comment:
I'm just looking very briefly at this. My only concern is if there are many student projects, like, more than 100. This `map` runs sequentially -- so that could take a lot of time if this `start_project` call is blocking.

OTOH, it would issue such start commands to all of them at once, basically, if it is non-blocking. 

Hmm, but I see, it's written in the same way as everywhere else. So, that should work then, too …